packages/sampler/stories/qa/Spinner.js

```diff
@@ -15,6 +17,38 @@ const
 		longText:'SpinnerWithLongText SpinnerWithLongText SpinnerWithLongText'
 	};
 
+class FocusOnSpinner extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			txt: '',
+			isLoading: false
+		};
+	}
+
+		handleChange = (e) => {
```

Comment:
Please fix indentation of these to be one tab, not two.

Reply:
Done